presentation/common/src/test/content/Descriptor.test.snap

```diff
@@ -1,5 +1,774 @@
+      "name": "Iran",
+    },
+  },
+  "connectionId": "f6ee137d-17d6-455b-bbed-ae0900eaf447",
+  "contentFlags": 0,
+  "contentOptions": "bar",
+  "displayType": "id eos quae",
+  "fields": Array [
+    Object {
+      "category": Object {
+        "description": "Accusantium ut doloremque quam quia reprehenderit libero.",
+        "expand": true,
+        "label": "Wooden",
+        "name": "Cambridgeshire",
+        "parent": undefined,
+        "priority": 96637,
+      },
+      "editor": Object {
+        "name": "bandwidth",
+      },
+      "isReadonly": true,
+      "label": "indigo",
+      "name": "Personal Loan Account",
+      "priority": 65570,
+      "renderer": Object {
+        "name": "custom_renderer",
+      },
+      "type": Object {
+        "typeName": "set",
+        "valueFormat": "Primitive",
+      },
+    },
+    Object {
+      "category": Object {
+        "description": "Accusantium ut doloremque quam quia reprehenderit libero.",
+        "expand": true,
+        "label": "Wooden",
+        "name": "Cambridgeshire",
+        "parent": undefined,
+        "priority": 96637,
+      },
+      "editor": undefined,
+      "isReadonly": true,
+      "label": "Technician Fish Ranch",
+      "name": "hacking",
+      "priority": 97576,
+      "renderer": Object {
+        "name": "custom_renderer",
+      },
+      "type": Object {
+        "typeName": "varchar",
+        "valueFormat": "Primitive",
+      },
+    },
+    Object {
+      "category": Object {
+        "description": "Accusantium ut doloremque quam quia reprehenderit libero.",
+        "expand": true,
+        "label": "Wooden",
+        "name": "Cambridgeshire",
+        "parent": undefined,
+        "priority": 96637,
+      },
+      "editor": Object {
+        "name": "Officer",
+      },
+      "isReadonly": true,
+      "label": "Generic Granite Bacon navigating",
+      "name": "Taiwan",
+      "priority": 78465,
+      "renderer": Object {
+        "name": "custom_renderer",
+      },
+      "type": Object {
+        "typeName": "bit",
+        "valueFormat": "Primitive",
+      },
+    },
+    Object {
+      "category": Object {
+        "description": "Qui magni consequatur est dolore nulla aperiam.",
+        "expand": true,
+        "label": "port bandwidth",
+        "name": "quantifying",
+        "parent": undefined,
+        "priority": 44020,
+      },
+      "editor": undefined,
+      "isReadonly": false,
+      "label": "Designer Nebraska AGP",
+      "name": "User-friendly",
+      "priority": 61507,
+      "properties": Array [
+        Object {
+          "property": Object {
+            "classInfo": "0x126b50000005c18",
+            "name": "Refined",
+            "type": "string",
+          },
+          "relatedClassPath": Array [
+            Object {
+              "isForwardRelationship": true,
+              "isPolymorphicRelationship": undefined,
+              "isPolymorphicTargetClass": undefined,
```

Comment:
Why do we get `undefined` here? They're not optional in `RelatedClassInfo` interface..